Form/AbstractModifySettingsForm.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Craue\ConfigBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2015 Christian Raue
+ * @license http://opensource.org/licenses/mit-license.php MIT License
+ */
+abstract class AbstractModifySettingsForm extends AbstractType {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function buildForm(FormBuilderInterface $builder, array $options) {
+		$builder->add('settings', Kernel::VERSION_ID < 20800 ? 'collection' : 'Symfony\Component\Form\Extension\Core\Type\CollectionType', array(
+			'type' => Kernel::VERSION_ID < 20800 ? 'craue_config_setting' : 'Craue\ConfigBundle\Form\Type\SettingType',
```

Comment:
So the deal is to infer from the existance of a method that I should pass FQCNs instead of deprecated aliases? I wouldn't call this feature detection because it's just a hacky way to find out which version of that component is used. :smirk:
Let's assume the `getBlockPrefix` method is removed some day, then this check would stop working, although using FQCNs might still be the right thing to do.